test/Interop/Cxx/templates/function-template-irgen-objc.swift

```diff
@@ -1,17 +0,0 @@
-// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
-
-// TODO: This needs to be fixed in the SwiftTypeConverter.
```

Comment:
Why is this test deleted?

Reply:
Same reason as the test was removed from the other test file: 
> This might look like a regression, but this should never have really been tested. It just happens to work. It was never designed, implemented, or meant to be supported.